features/heading-selectors.yml

```
@@ -0,0 +1,7 @@
+name: Heading pseudo-classes
+description: The `:heading` and `:heading()` CSS pseudo-classes match headings of varying levels. The `:heading` pseudo-class matches any heading (`<h1>` through `<h6>`). The `:heading(<integer>#)` selector matches any of the given levels, for example `:heading(1, 2)` will match headings level 1 and 2.
```

Tiny nits on the description (to avoid future tense, to avoid non-literal code, and align the example with other descriptions), but like this a lot. Thank you!

```suggestion
description: The `:heading` and `:heading()` CSS pseudo-classes match headings of varying levels. The `:heading` pseudo-class matches any heading (`<h1>` through `<h6>`). The `:heading()` selector matches any of the given levels. For example, `:heading(1, 2)` matches headings level 1 and 2.
```